addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightV3White.java

```diff
@@ -0,0 +1,186 @@
+            for (int i = 0; i < repeatCount; i++) {
+                changeColorTemperature(1, state);
+            }
+        } else if (newLevel < oldLevel) {
+            for (int i = 0; i < repeatCount; i++) {
+                changeColorTemperature(-1, state);
+            }
+        }
+
+        state.colorTemperature = color_temp;
+    }
+
+    @Override
+    public void changeColorTemperature(int color_temp_relative, MilightThingState state) {
+        int newPercent = Math.min(100, Math.max(state.colorTemperature + color_temp_relative, 0));
+        state.colorTemperature = newPercent;
+        sendQueue.queue(
+                QueuedSend.NO_CATEGORY, new byte[] { (byte) (color_temp_relative > 0 ? 0x3E : 0x3F), 0x00, 0x55 });
+    }
+
+    // This just emulates an absolute brightness command with the relative commands.
+    @Override
+    public void setBrightness(int value, MilightThingState state) {
+        if (value <= 0) {
+            setPower(false, state);
+            return;
+        } else if (value >= 100) {
+            setFull(zone, state);
+            return;
+        }
+
+        // White Bulbs: 11 levels of brightness + Off.
+        final int newLevel = (int) Math.ceil((value * brightnessLevels) / 100.0);
+
+        // When turning on start from full brightness
+        int oldLevel;
+        if (state.brightness == 0) {
+            setFull(zone, state);
+            oldLevel = brightnessLevels;
+        } else {
+            oldLevel = (int) Math.ceil((state.brightness * brightnessLevels) / 100.0);
+        }
+
+        final int repeatCount = Math.abs(newLevel - oldLevel);
+        logger.debug("milight: dim from '{}' with command '{}' via '{}' steps.", String.valueOf(state.brightness),
+                String.valueOf(value), repeatCount);
+        if (newLevel > oldLevel) {
+            for (int i = 0; i < repeatCount; i++) {
+                changeBrightness(+1, state);
+            }
+        } else if (newLevel < oldLevel) {
+            for (int i = 0; i < repeatCount; i++) {
+                changeBrightness(-1, state);
+            }
+        }
+
+        state.brightness = value;
+    }
+
+    @Override
+    public void changeBrightness(int relative_brightness, MilightThingState state) {
+        if (relative_brightness == 0) {
+            return;
+        }
+        state.brightness = Math.max(state.brightness + relative_brightness, 0);
+
+        if (state.brightness == 0) {
+            setPower(false, state);
+        } else {
+            setPower(true, state);
+            sendQueue.queue(
+                    QueuedSend.NO_CATEGORY, new byte[] { (byte) (relative_brightness < 0 ? 0x34 : 0x3C), 0x00, 0x55 });
+        }
+    }
+
+    @Override
+    public void changeSpeed(int relative_speed, MilightThingState state) {
+
+    }
+
+    private final static byte prev_animation_mode[] = { 0x27, 0x00, 0x55 };
+
+    @Override
+    public void previousAnimationMode(MilightThingState state) {
```

Comment:
There's nothing on the white bulb remote or on the easybulb app for animation for white bulbs